api/src/pcapi/core/offerers/api.py

```diff
@@ -268,12 +268,29 @@ def update_venue_location(  # pylint: disable=too-many-positional-arguments
     for the API.
     """
     assert venue.offererAddress is not None
-    street = location_modifications.get("street") or venue.offererAddress.address.street
-    city = location_modifications.get("city") or venue.offererAddress.address.city
-    postal_code = location_modifications.get("postalCode") or venue.offererAddress.address.postalCode
-    latitude = location_modifications.get("latitude") or venue.offererAddress.address.latitude
-    longitude = location_modifications.get("longitude") or venue.offererAddress.address.longitude
-    ban_id = location_modifications.get("banId") or venue.offererAddress.address.banId
+    # When street is cleared from the BO, location_modifications contains: {'street': None}
+    street = (
+        location_modifications["street"] if "street" in location_modifications else venue.offererAddress.address.street
+    )
```

Comment:
j'aurai fait ça pour simplifier :
```suggestion
    street = location_modifications.get("street", venue.offererAddress.address.street)
```

Reply:
Oui bien sûr, j'ai vu trop compliqué ! Je modifie.